frontend/simple/model/state.js

```diff
@@ -194,6 +171,23 @@ const getters = {
       if (!groupId) return 0
       return Object.keys(state[groupId].profiles).length
     }
+  },
+  proposalData (state, getters) {
+    return (proposalHash, groupId) => {
+      if (!groupId) groupId = state.currentGroupId
+      if (!groupId) return null
+      const groupData = getters.currentGroupState
+      const userData = getters.currentUserIdentityContract.attributes
+      const proposal = state[groupId].proposals[proposalHash]
+      return {
+        ...proposal,
+        hash: proposalHash,
+        voterCount: Object.entries(groupData.profiles).length,
+        fromWhat: groupData[proposal.doWhat] || null,
+        myVote: proposal.votes.find(vote => vote.username === userData.name) || 0,
+        isMyProposal: proposal.whoProposed === userData.name
+      }
+    }
```

Comment:
This function is unnecessary. I think it should be deleted. It is generally bad mojo to create objects randomly like this. Instead, it is better to use the raw data of objects that are well defined, like the ones in `group.js`, and apply functions to them.

So I recommend deleting the `proposalData` getter.